code-studio/src/js/videos.js

```diff
@@ -228,17 +234,33 @@ function setupVideoFallback(videoInfo, playerWidth, playerHeight, shouldStillAdd
     return;
   }
 
-  videos.onYouTubeBlocked(function () {
-    if (!shouldStillAddCallback()) {
-      return;
-    }
-    addFallbackVideoPlayer(videoInfo, playerWidth, playerHeight);
-  });
+  videos.onYouTubeBlocked(
+      function () {
```

Comment:
2 spaces per indentation, and i think `function() {` goes on same line as `onYouTubeBlocked`

style guide is at https://github.com/code-dot-org/code-dot-org/blob/staging/STYLEGUIDE.md#javascript